src/devices/src/virtio/net/device.rs

```diff
@@ -195,7 +229,7 @@ impl Net {
             activate_evt: EventFd::new(libc::EFD_NONBLOCK).map_err(Error::EventFd)?,
             config_space,
             mmds_ns: None,
-            guest_mac: guest_mac.copied(),
+            guest_mac: mac_addr,
```

Comment:
guest_mac no longer needs to be an option, right?

Reply:
It's no longer an `Option` in `Net`